### PR TITLE
init script to auto start kvm network at boot

### PR DESCRIPTION
**Pull request by ejannett**
When user creates creates a Libvirt network we also create an initd script 
to make the network available after reboot.

test done :
1 - create a libvrit network
 * check the network is up 
 * check that init script is in place
 * check that after reboot network is also up

test in progress :
 create a network , create a guest then test connectivity before and after a reboot

### CODE REVIEW

**Comment by ejannett** [comment]
test of the reboot completed
1 - create a network
2 - create a guest then test connectivity to outside world 
3 - reboot the host
4 - test connectivity of the guest to outside world

File: setup.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def run(self):
 
 setup(
     name="oci-utils",
-    version="0.10.0",
+    version="0.10.1",
     author="Laszlo Peter, Qing Lin",
```
Comment (wbora) [inline setup.py]: Please add Guido and your name here as well 😃

File: libexec/oci-kvm-network-script [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+stop() {
+   /usr/bin/virsh net-destroy --network ${__KVM_NETWORK_NAME__}
+   return $?
+}
+
+status() {
+    /usr/bin/virsh net-info --network ${__KVM_NETWORK_NAME__}
+    /usr/bin/virsh net-dhcp-leases --network ${__KVM_NETWORK_NAME__}
+}
+
```
[the remark this answers is not in the document]
Comment (ejannett) [reply libexec/oci-kvm-network-script]: Not sure to understand the request

**Comment by ejannett** [comment]
Testing done:

- run functional tests  
- create a network : check that the service is up , build a guest and verify connectivity 
- delete  a network : check the service is down and service unit file is gone

File: libexec/oci-kvm-network-script (hunk 2 above)
[the remark this answers is not in the document]
Comment (wbora) [reply libexec/oci-kvm-network-script]: Something like:
```
...

if [[ -z "${__VNIC_DEFAULT_GW__}" ]] || \
   [[ -z "${__NET_DEV__}" ]] || \
   [[ -z "${__RT_TABLE_NAME__}" ]] || \
   ...
then
    echo "ERROR: Environment not set correctly!" >&2
    exit 1
fi

case "$1" in
...
```

Idea is to make sure we do not blindly run networks commands we have no idea what the effect would be..

**Pull request merged**